packages/lesswrong/lib/crud/withMulti.ts

```diff
@@ -198,7 +199,7 @@ export function useMulti<
   // if showLoadMore returned true.
   const showLoadMore = alwaysShowLoadMore || (enableTotal ? (count < totalCount) : (count >= effectiveLimit));
   
-  const loadMore: LoadMoreCallback = async (limitOverride?: number) => {
+  const loadMore: LoadMoreCallback = useStabilizedCallback(async (limitOverride?: number) => {
```

Comment:
Is this because the reference to `loadMore` would change under some circumstances, but the actual effect it'd have wouldn't?

Reply:
(I would assume you'd in fact want it to change when the limit changed, but I don't know exactly what's going on here)

Reply:
`loadMore` changes every time. What `useStabilizedCallback` does is store the last-seen version in a ref, and return a stable wrapper function which calls the last-seen version. This isn't safe for anything that could be called from inside a render function, but it _is_ safe for things called exclusively from event handlers (which `loadMore` is).